api/MediaDeviceInfo.json

```diff
@@ -25,11 +25,11 @@
             "version_added": false
           },
           "opera": {
-            "version_added": false,
+            "version_added": "42",
             "notes": "This interface can be used in Opera by using the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill."
```

Comment:
Let's delete this note now that it is supported.

Reply:
These changes weren't supposed to be in this PR, oops!  Cherry-picking.